### PR TITLE
- Split main css styles into seperate css files

### PR DESCRIPTION
**Pull request by abbasCoder51**
## PR Checklist

Please check all that apply to this PR using "x":

- [x] I have checked that this PR is not a duplicate of an existing PR (open, closed or merged)
- [ ] I have checked that this PR does not introduce a breaking change
- [ ] This PR introduces breaking changes and I have provided a detailed explanation below

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting)
- [x] Refactoring (no functional changes)
- [ ] Documentation changes
- [ ] Other - Please describe:

## Fixes

Issue Number: #140

### CODE REVIEW

File: src/css/base.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
+@import url('theme.css');
```
Comment (btzr-io) [inline src/css/base.css]: the build should compile all this into a single file, how does the `@import` works in this case with webpack and rollup ?

Comment (btzr-io) [reply src/css/base.css]: Well looks like it works for webpack, not sure about rollup: https://github.com/webpack-contrib/css-loader#import

Comment (btzr-io) [reply src/css/base.css]: can you test this on rollup build ?

File: src/index.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ import CanvasRender from '@/components/render'
 import { ReaderContext, ReaderProvider } from '@/context'
 
 // Css
-import './css/styles.css'
+import './css/base.css'
```
Comment (btzr-io) [inline src/index.js]: rename to `main.css` or `index.css` also use: `@/css/file.css`

Comment (abbasCoder51) [reply src/index.js]: Changed 'base.css' to 'main.css', I also tried to change it using the format '@/css/main.css' but it breaks the build

File: src/css/base.css (hunk 1 above)
Comment (abbasCoder51) [reply src/css/base.css]: what command do I need to run to test this build?

File: src/index.js (hunk 2 above)
Comment (btzr-io) [reply src/index.js]: Ok, I'll look into it. its fine like that thanks :v:

File: src/css/base.css (hunk 1 above)
Comment (btzr-io) [reply src/css/base.css]: I know webpack supports this syntax:
```
@import url('theme.css')  and @import 'theme.css'
```
Can you confirm rollup also support it ? the command is `yarn build`

Comment (abbasCoder51) [reply src/css/base.css]: Tested with rollup, breaks the build if i use @import inside a css file, imported all relevant css files in index.js using import './css/<css-name>'

Comment (btzr-io) [reply src/css/base.css]: Ok, too bad `rollup` don't support that syntax :disappointed:

**Comment by btzr-io** [comment]
I did some changes to the css file, could you update this ?

**Comment by btzr-io** [comment]
Should I close this ?

**Pull request closed without merging**